pkg/cmd/attestation/verification/embed/tuf-repo.github.com/root.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
1. What is the lifecycle of these signatures?
2. What events might happen that require this to be updated and pushed out?
3. What problems might users face if they are using an older version?
4. Does `gh attestation` tell the user these might be out of date and to go run some command to get the most current version?

Reply:
How much of this should be publicly documented in `docs`?

Reply:
1. We expect the signatures to be updated every six months
2. Once the new version of the root is available every six months, I would need to open a pull request to update the file
3. If the user has an older version downloaded, the TUF root client within the command set will query for a newer version of the root and use that one
4. Yes, gh-attestation will grab the current version

Let me get back to you later today on what should be documented.